params.go

```diff
@@ -376,6 +376,61 @@ func BuildQueryString(opts interface{}) (*url.URL, error) {
 	return nil, fmt.Errorf("options type is not a struct")
 }
 
+type URL struct {
+	u *url.URL
+}
+
+func (u *URL) String() string {
+	return u.u.String()
+}
+
+type URLBuilder struct {
```

Comment:
please add docstring for exported methods

Reply:
done